pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -312,6 +319,15 @@ func (r *reconciler) ensureLoadBalancerService(ci *operatorv1.IngressController,
 	return true, currentLBService, nil
 }
 
+// Determines whether a service is owned by an ingress controller.
+func isServiceOwnedByIngressController(service *corev1.Service, ci *operatorv1.IngressController) bool {
+	if service == nil || service.Labels[manifests.OwningIngressControllerLabel] != ci.Name {
+		return false
+	} else {
+		return true
+	}
```

Comment:
Slightly more idiomatic Go:
```suggestion
	if service != nil || service.Labels[manifests.OwningIngressControllerLabel] == ci.Name {
		return true
	}
	return true
```

Reply:
TIL GO this style. checks out :+1: 

Got it - `||` should be `&&` but I understand.

Reply:
TIL GO this style. checks out :+1: 

Got it - `||` should be `&&` but I understand.

Reply:
Done